### PR TITLE
Temporarily Removing CoC and references to audacity contributing advice

### PR DESCRIPTION
**Pull request by caughtquick**
Resolves: *(direct link to the issue)*

*Files that are inaccurate for this fork and may be added later when decided on*

- [x] I made sure the code compiles on my machine
- [x] I made sure there are no unnecessary changes in the code
- [x] I made sure the title of the PR reflects the core meaning of the issue you are solving
- [x] I made sure the commit message(s) contain a description and answer the question "Why do those changes fix that particular issue?" or "Why are those changes really necessary as improvements?"

### CODE REVIEW

**Review by oibind — COMMENTED** [review]
Why remove the Contributor Covenant Code of Conduct?

**Comment by caughtquick** [comment]
> Why remove the Contributor Covenant Code of Conduct?

I think @cookiengineer wants a CoC but I just removed it until a CoC that is decided by the community and cookiengineer can be chosen rather than this just being the on that is selected since it came from audacity/audacity.

**Comment by oibind** [comment]
Not to be too pedantic but the title of PR doesn't accurately reflect the full scope of what was removed/changed. ☺️😊

**Comment by caughtquick** [comment]
> Not to be too pedantic but the title of PR doesn't accurately reflect the full scope of what was removed/changed. ☺️😊

What would you suggest as a better title, I'll happily change it if it's less misleading

**Comment by oibind** [comment]
Perhaps: "Removing CoC and references to audacity contributing advice"

Something like that. Just more descriptive I guess.

**Review by oibind — APPROVED** [review]

**Comment by n0toose** [comment]
Hey, thanks for the PR and apologies for removing `CONTRIBUTING.md` without having seen this pull request before.

- The PR template seems just fine for the time being. Apart from a grammar mistake, taking things from the official Audacity project that were meant to keep things in order seems like a good idea to me and it doesn't make much sense to get rid of that.
- I feel like removing the Code of Conduct is as much of a drastic change as including one. As in, we have to agree that "we won't do this the way Audacity does" or "we will keep things as they are".
- Already dealt with `CONTRIBUTING.md` myself.

**Comment by caughtquick** [comment]
> Hey, thanks for the PR and apologies for removing `CONTRIBUTING.md` without having seen this pull request before.
> 
> 
> 
> - The PR template seems just fine for the time being. Apart from a grammar mistake, taking things from the official Audacity project that were meant to keep things in order seems like a good idea to me and it doesn't make much sense to get rid of that.
> 
> - I feel like removing the Code of Conduct is as much of a drastic change as including one. As in, we have to agree that "we won't do this the way Audacity does" or "we will keep things as they are".
> 
> - Already dealt with `CONTRIBUTING.md` myself.

Fair points, I'll take a look and see what I should, or should not delete tomorrow for the rest of this PR if that's alright with you

**Comment by n0toose** [comment]
Yeah, we distanced ourselves from Audacity because we disagreed with their *recent* direction and their general pattern of miscommunications. We should avoid following the same mistakes, however, we should not burn bridges and disregarding other things without good reason and we must show respect the work of all of the contributors that have worked on Audacity over the past decades. Should we *adapt* what they have done? Yup, sure, but let's not suggest something that's destructive (such as "writing Audacity from scratch").

**Comment by n0toose** [comment]
Nevertheless, feel free to keep this PR up and make the changes you wish to make. I'm going to take a look either way! :D

Thanks again for your work!

**Comment by caughtquick** [comment]
Yeah I personally agree with a code of conduct but I felt it was better to not have a code of conduct until one was decided rather than vice versa but I have no absolute personal opinion either way and completely understand where you are coming from too.

**Comment by macginitie** [comment]
Imo, having a CoC is fine, this one seems mostly fine, but I think y'all should definitely remove the email link in it: "feedback@audacity.com"... which is maybe not fine?

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -1,46 +0,0 @@
...
-* The use of sexualized language or imagery and unwelcome sexual attention or advances
-* Other conduct which could reasonably be considered inappropriate in a professional setting
-
-## Our Responsibilities
-
-Project maintainers are responsible for clarifying the standards of acceptable behavior and are expected to take appropriate and fair corrective action in response to any instances of unacceptable behavior.
-
-Project maintainers have the right and responsibility to remove, edit, or reject comments, commits, code, wiki edits, issues, and other contributions that are not aligned to this Code of Conduct, or to ban temporarily or permanently any contributor for other behaviors that they deem inappropriate, threatening, offensive, or harmful.
-
-## Scope
```
Comment (ashley062190) [inline CODE_OF_CONDUCT.md:31]: Opt out always

**Pull request closed without merging**